.github/workflows/vercel_push.yml

```diff
@@ -0,0 +1,30 @@
+name: Vercel -> Algolia Crawler (push on main)
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  algolia_reindex:
+    name: Algolia Reindex
```

Comment:
Just thought about that, we publicly only communicate on "Crawl" and "Recrawl", to avoid confusion with the suffix index, so you should replace by "Recrawl"